core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -34,17 +34,19 @@ oppia.directive('collectionSummaryTile', [
         getThumbnailIconUrl: '&thumbnailIconUrl',
         getThumbnailBgColor: '&thumbnailBgColor',
         isLinkedToEditorPage: '=?isLinkedToEditorPage',
-        getCategory: '&category'
+        getCategory: '&category',
+        isPlaylistMode: '&playlistMode'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/summary_tile/' +
         'collection_summary_tile_directive.html'),
       controller: [
         '$scope', 'oppiaDatetimeFormatter',
         'COLLECTION_VIEWER_URL', 'COLLECTION_EDITOR_URL', function($scope,
-        oppiaDatetimeFormatter, COLLECTION_VIEWER_URL,
-        COLLECTION_EDITOR_URL) {
+          oppiaDatetimeFormatter, COLLECTION_VIEWER_URL,
+          COLLECTION_EDITOR_URL) {
           $scope.DEFAULT_EMPTY_TITLE = 'Untitled';
+          $scope.collectionType = constants.ACTIVITY_TYPE_COLLECTION;
```

Comment:
This is a constant, so call it $scope.ACTIVITY_TYPE_COLLECTION

Reply:
Done!